src/chains/serializers.py

```diff
@@ -28,7 +28,7 @@ class GasPriceFixed1559Serializer(serializers.Serializer[GasPrice]):
 
 
 class GasPriceSerializer(serializers.Serializer[GasPrice]):
-    def to_representation(self, instance: GasPrice) -> ReturnDict:
+    def to_representation(self, instance: GasPrice) -> ReturnDict:  # type: ignore
```

Comment:
Isn't there any other "fix" besides ignoring the type?

Reply:
I updated the serializer type from `serializers.Serializer` to `serializer.ModelSerializer` in 946871f9cd95aabc19823ed641c37ba54481d3a2 and added the `Meta` accordingly. Please lmk what you think.

Reply:
I think it's a good solution! 👏